src/core/lib/security/transport/server_auth_filter.cc

```diff
@@ -228,6 +251,7 @@ static grpc_error* init_call_elem(grpc_call_element* elem,
   GRPC_CLOSURE_INIT(&calld->recv_trailing_metadata_ready,
                     recv_trailing_metadata_ready, elem,
                     grpc_schedule_on_exec_ctx);
+  calld->seen_recv_trailing_ready = false;
```

Comment:
Not necessary, since filter member is initialized to all zero.

Reply:
done!